Code/Tools/LevelEdit/MainDialogBar.cpp

```diff
@@ -202,21 +202,25 @@ MainDialogBarClass::Initialize (void)
 {
 	UpdateData (FALSE);
 	
+	char presets_string[] = "Presets";
 	TC_ITEM tab_info = { 0 };
 	tab_info.mask = TCIF_TEXT;
-	tab_info.pszText = "Presets";
+	tab_info.pszText = presets_string;
```

Comment:
I haven't take a close look at this code before, but would this cause issues?  I believe that the original code would give a pointer to a string in constant memory, while the new code points to a string on the stack.  I don't know if the value of the string is used after the method is complete.

Reply:
`TC_ITEM::pszText` has type `LPSTR` instead of `LPCSTR`. This is a const correctness fix.